contracts/finality/src/exec/public_randomness.rs

```diff
@@ -79,6 +79,7 @@ pub fn handle_public_randomness_commit(
         PubRandCommit {
```

Comment:
`validate_pub_rand_commit` which is above still have:
```rust
    // Check for overflow when doing (StartHeight + NumPubRand)
    // To avoid public randomness reset
    let end_height = start_height.saturating_add(num_pub_rand);
    if start_height >= end_height {
        return Err(ContractError::OverflowInBlockHeight {
            start_height,
            end_height,
        });
    }
```

to check for overlfow. This should be adjusted